src/components/Map.js

```diff
@@ -1,19 +1,20 @@
 import React, { useRef } from 'react';
-import PropTypes from 'prop-types';
+import PropTypes, { any } from 'prop-types';
 import { Map as BaseMap, TileLayer, ZoomControl } from 'react-leaflet';
 
 import { useConfigureLeaflet, useMapServices, useRefEffect } from 'hooks';
 import { isDomAvailable } from 'lib/util';
 
 const DEFAULT_MAP_SERVICE = 'OpenStreetMap';
 
-const Map = ( props ) => {
-  const { children, className, defaultBaseMap = DEFAULT_MAP_SERVICE, mapEffect, ...rest } = props;
-
-  const mapRef = useRef();
+const Map = React.forwardRef(( props ) => {
```

Comment:
the ref argument actually needs to be included as a second argument next to props

https://reactjs.org/docs/forwarding-refs.html

if you look at the console when running the project, its throwing an error

![image](https://user-images.githubusercontent.com/1045274/97245885-66312580-17d2-11eb-863c-cdee512a8f2d.png)